app/models/users/user.rb

```diff
@@ -32,6 +34,13 @@ def self.check_or_create_token(user, access_token)
     user
   end
 
+  def self.check_or_create_image(user, image)
+    unless user.google_image
+      user.update_attributes(google_image: image)
```

Comment:
this wont update if the image on google has changed. we should probably check the url and if its different, also update it.